modules/standard/IO.chpl

```diff
@@ -3981,8 +3980,10 @@ proc channel.writeln(out error:syserr):bool {
 
 // documented in style= error= version
 pragma "no doc"
-proc channel.writeln():bool {
-  return this.write(new ioNewline());
+proc channel.writeln():bool throws {
+  try {
+    return this.write(new ioNewline());
+  }
```

Comment:
In the initial design phase I thought of `try` as being used on statements rather than expressions, but this might be a distinction without difference and I would prefer this form to `try return g();`. I've put it under the everything else section in #4977.